scripts/package-build

```diff
@@ -228,7 +228,7 @@ SCRIPT
             for step in pre post; do
                 if [[ $(type -t "$step"upgrade) = function ]] \
                     || [[ $(type -t "$step"remove) = function ]] \
-                    || [ -d "$pkgdir"/opt/usr/share/applications ] && [ $step = post ]; then
+                    || { [ -d "$pkgdir"/opt/usr/share/applications ] && [ $step = post ]; }; then
```

Comment:
Well spotted. Sorry I missed that during my review.

Reply:
I had it in `()` originally, but shellcheck didn't like it, and I misunderstood how bash does it's boolean logic.